examples/parse_api/src/app/controllers/about.controller.ts

```diff
@@ -64,6 +65,16 @@ export class AboutController extends ApiController {
         };
     }
 
+    // API: /toBoolean/true - should work
+    // API: /toBoolean/abc - throws exception
+    @HttpGet('/toBoolean/:id')
+    toBoolean2(@QueryParam(toBoolean) id: boolean): any {
+        return {
+            id: id,
+            msg: 'Built-in Converter to boolean'
+        };
+    }
+
```

Comment:
Can you please test this cases as well?

@HttpGet('/toBoolean/:id')
toBoolean2(@Parse(toBoolean) id: boolean): any {
   return {
              id: id,
              msg: 'Built-in Converter to boolean'
     };
}

Reply:
Yes.